src/Routes/WizardPage/WizardPage.tsx

```diff
@@ -43,6 +44,13 @@ const WizardPage = () => {
   const linkLearnMoreAbout = 'https://access.redhat.com/articles/1586893';
   const linkLearnMoreAboutRemovingDirectoryAndDomainServices = 'https://access.redhat.com/articles/1586893';
 
+  useEffect(() => {
+    if (!appContext.rbac.permissions.hasTokenCreate || !appContext.rbac.permissions.hasDomainsUpdate) {
```

Comment:
This needs to be `if (!rbac.isLoading && (!rbac.permissions.hasTokenCreate || !rbac.permissions.hasDomainsUpdate)) {` so that it doesnt redirect before having it loaded.

Reply:
Being fixed in https://github.com/podengo-project/idmsvc-frontend/pull/73